src/components/application_manager/test/commands/mobile/set_global_properties_test.cc

```diff
@@ -156,14 +164,15 @@ class SetGlobalPropertiesRequestTest
     EXPECT_CALL(mock_message_helper_, VerifyImageVrHelpItems(_, _, _)).Times(0);
     EXPECT_CALL(app_mngr_,
                 RemoveAppFromTTSGlobalPropertiesList(kConnectionKey));
-    SmartObject vr_help_title("title");
-    EXPECT_CALL(*mock_app_, vr_help_title()).WillOnce(Return(&vr_help_title));
     EXPECT_CALL(*mock_app_, set_help_prompt(help_prompt));
     EXPECT_CALL(*mock_app_, help_prompt()).WillOnce(Return(&help_prompt));
     EXPECT_CALL(*mock_app_, set_timeout_prompt(timeout_prompt));
     EXPECT_CALL(*mock_app_, timeout_prompt()).WillOnce(Return(&timeout_prompt));
     EXPECT_CALL(*mock_app_, app_id()).WillOnce(Return(kAppId));
 
+    ON_CALL(*mock_app_, help_prompt_manager())
```

Comment:
@SNiukalov please use `EXPECT_CALL` instead of `ON_CALL` in local unit tests

Reply:
Thanks, fixed in https://github.com/smartdevicelink/sdl_core/pull/2186/commits/75fec9acfe8442742e0b6699a651eb564f307868